<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EATC BANK DEPOSIT PAGE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Please Indicate the Amount You Want to Insert:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 26pt &quot;Matura MT Script Capitals&quot;;</string>
   </property>
   <property name="text">
    <string>EACT BANK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returntopreviouspage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Return to Previous Screen</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_pas">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushentry">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Entry</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enty">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
